plugins/woocommerce/client/admin/client/launch-your-store/hub/sidebar/tasklist.tsx

```diff
@@ -126,11 +129,15 @@ export const CompletedTaskItem: React.FC< {
 	</SidebarNavigationItem>
 );
 
-export const IncompleteTaskItem: React.FC< {
+export const IncompleteTaskItem = ( {
+	task,
+	classNames,
+	onClick,
+}: {
 	task: TaskType;
 	classNames?: string;
 	onClick: () => void;
-} > = ( { task, classNames, onClick } ) => (
+} ) => (
```

Comment:
FYI other reviewers who like me, may have thought this looked like a change to a default assignment - no it's not.